apps/login/src/lib/server/password.ts

```diff
@@ -192,24 +213,14 @@ export async function sendPassword(command: UpdateSessionCommand) {
     }
 
     return { redirect: `/mfa?` + params };
-  } else if (user.state === UserState.INITIAL) {
-    const params = new URLSearchParams({
-      loginName: session.factors.user.loginName,
-    });
-
-    if (command.authRequestId) {
-      params.append("authRequestId", command.authRequestId);
-    }
-
-    if (command.organization || session.factors?.user?.organizationId) {
-      params.append(
-        "organization",
-        command.organization ?? session.factors?.user?.organizationId,
-      );
-    }
-
-    return { redirect: `/password/change?` + params };
-  } else if (command.forceMfa && !availableSecondFactors.length) {
+  }
+  // TODO: check if handling of userstate INITIAL is needed
```

Comment:
I would say no, as it does not exist on v2 api users

Reply:
agree. I keep the comments for the moment though as they help to identify the use cases where the user state is critical